### PR TITLE
Display orders on stocks

### PR DESCRIPTION
**Pull request by akampjes**
Each stock should display the orders (and order info) on it.
Orders should be ordered in date descending order and publicly
viewable.

### CODE REVIEW

**Comment by akampjes** [comment]
@Widdershin what do you think?

File: spec/features/order_spec.rb [hunk 1]
```diff
@@ -22,4 +20,11 @@
 
     expect(current_path).to eq orders_path
   end
+
+  scenario 'Publically display orders' do
+    order = create(:buy_order, quantity: 1337)
+
+    visit stock_path(order.stock)
+    expect(page).to have_content '1337'
```
Comment (Widdershin) [inline spec/features/order_spec.rb]: I would suggest you add a newline above the `expect`.

It's nice to have some visual separation between the arrange, act and assert sections of the spec.

File: spec/models/stock_spec.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Stock, type: :model do
-  it 'is vaild with a name and symbol' do
+  it 'is valid with a name and symbol' do
```
Comment (Widdershin) [inline spec/models/stock_spec.rb]: :+1:

**Comment by akampjes** [comment]
Next question then is, do we just add new commits or `gc --amend` and `gp -f`?

**Comment by Widdershin** [comment]
Personally I vote for new commits. Some projects prefer to alter time for the sake of having a "clean" git log, but I would rather capture all the work that went it and how it happened.

http://paul.stadig.name/2010/12/thou-shalt-not-lie-git-rebase-ammend.html

What do you think?

**Comment by akampjes** [comment]
In the past, projects I've worked on haven't made good use of merges and have preferred to rebase for a linear history.

In the case of whitespace changes to this patch, I would amend and forcepush (if I'm working on this branch only by myself). I don't need a commit that just says `Add newline before expect`, that doesn't add any value to my history.

However if I discovered a change to the business logic later, I think making a new commit would be more appropriate.

I think I want to start using merges more so anyone can see how features have evolved.

**Pull request merged**